### PR TITLE
yashim/text: update footer text (EU) and regulatory info

### PR DESCRIPTION
**Pull request by yashim-deriv**
Co-authored-by: Yashim Wong <yashimwong@gmail.com>

# Description

Summary
- Text change on footer (EU) and regulatory info page

Changes

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/layout/footer.js [hunk 1]
```diff
@@ -568,7 +568,7 @@ const Footer = ({ type }) => {
                             </DisclaimerParagraph>
                             <DisclaimerParagraph>
                                 <Localize
-                                    translate_text="Deriv (MX) Ltd, Millennium House, Level 1, Victoria Road, Douglas IM2 4RW, Isle of Man, is licensed and regulated in Great Britain by the Gambling Commission under <1>account no. 39172</1> and by the Gambling Supervision Commission in the Isle of Man (<0>view licence</0>)."
+                                    translate_text="Deriv (MX) Ltd, Millennium House, Level 1, Victoria Road, Douglas IM2 4RW, Isle of Man, licensed and regulated by the Gambling Supervision Commission in the Isle of Man (<0>view licence</0>) and by the UK Gambling Commission for clients in the UK (<1>account no. 39172</1>)."
```
Comment (mustofa-binary) [inline src/components/layout/footer.js:571]: the link is the other way round

Comment (yashim-deriv) [reply src/components/layout/footer.js:571]: seems correct since they swap the account with licence position

**Review by sean-binary — APPROVED** [review]

File: src/components/layout/footer.js [hunk 2]
```diff
@@ -664,9 +665,8 @@ const Footer = ({ type }) => {
...
-                                            CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. 71% of retail investor accounts lose money when trading CFDs with this provider. You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money.<0/><0/>
-                                            The products mentioned here may be affected by changes in currency exchange rates. If you invest in these products, you may lose some or all of your investment and the value of your investment may fluctuate. You should never invest money that you cannot afford to lose and never trade with borrowed money."
+                                            translate_text="The financial products offered via this website include digitals, contracts for difference (CFDs), and other complex derivatives and financial products. Trading financial products may not be suitable for everyone. CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage.  74% of retail investor accounts lose money when trading CFDs with Deriv Investments (Europe) Ltd. <0></0>
```
Comment (AbdallaMourad) [inline src/components/layout/footer.js:668]: We can replace `<0></0>` with `<0/>`

File: src/pages/regulatory/index.js [hunk 3]
```diff
@@ -223,7 +232,7 @@ const Regulatory = () => {
                                     </Desktop>
                                     <Text mt="0.8rem" mb="1.6rem" max_width="58.8rem">
                                         <Localize
-                                            translate_text="Deriv (Europe) Limited, W Business Centre, Level 3, Triq Dun Karm, Birkirkara BKR 9033, Malta, is licensed for synthetic indices by the Malta Gaming Authority (<0>licence no. MGA/B2C/102/2000</0>), by the Gambling Commission for clients in Great Britain under (<1>account no. 39495</1>), and by the Revenue Commissioners for clients in Ireland (licence no. 1010285)."
+                                            translate_text="Deriv (Europe) Limited, W Business Centre, Level 3, Triq Dun Karm, Birkirkara BKR 9033, Malta, is licensed for synthetic indices by the Malta Gaming Authority (<0>licence no. MGA/B2C/102/2000</0>), by the Gambling Commission for clients in Great Britain under <1>account no. 39495</1>, and by the Revenue Commissioners for clients in Ireland (licence no. 1010285)."
```
Comment (AbdallaMourad) [inline src/pages/regulatory/index.js:235]: does the last licence need a URL?

Comment (yashim-deriv) [reply src/pages/regulatory/index.js:235]: I just confirmed with compliance and u r right, thanks! really good catch

**Review by suthesan-deriv — APPROVED** [review]

**Review by AbdallaMourad — APPROVED** [review]

**Pull request merged**